src/osbuild-manifests/aws_tag.py

```diff
@@ -0,0 +1,59 @@
+import os
+import subprocess
+import json
+import argparse
+
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--stream', dest='stream', type=str, help='Fedora stream', required=True)
+    parser.add_argument('--arch', dest='arch', type=str, help='Architecture', default='x86_64')

```

Comment:
for what we are doing we actually want x86_64 and aarch64.. 

Probably the easier thing to do here is to not limit this by arch and just pass in `--arch=all` to `cosa buildfetch` then iterate over the arches that exist for that build.